prac_08/box_layout.kv

```diff
@@ -0,0 +1,16 @@
+BoxLayout:
+    orientation: 'vertical'
+#    orientation: 'horizontal'
+    Button:
+        text: 'Clear'
+        on_press: app.handle_clear()
+    TextInput:
+        id: input_name
+        text: ''
+    Button:
+        text: 'Greet'
+        on_press: app.handle_greet()
+    Label:
+        id: output_label
+        text: 'Enter your name'
+        color: (1, 1, 0, 1)
```

Comment:
Just a few of the files have a minor error here, there is meant to be an empty line at the end of the file